src/test/ui/mismatched_types/issue-74918-missing-lifetime.stderr

```diff
@@ -0,0 +1,30 @@
+error[E0106]: missing lifetime specifier
+  --> $DIR/issue-74918-missing-lifetime.rs:9:31
+   |
+LL |     type Item = IteratorChunk<T, S>;
+   |                               ^ expected named lifetime parameter
+   |
+help: consider introducing a named lifetime parameter
+   |
+LL |     type Item<'a> = IteratorChunk<<'a>T, S>;
```

Comment:
do we already have an issue for this error message?

Reply:
I'm not sure, but to fix it we need to extend the logic in the last `match` at the end of `add_missing_lifetime_specifiers_label`.

Reply:
https://github.com/rust-lang/rust/pull/75372